lib/slack/src/index.ts

```diff
@@ -15,4 +12,40 @@ app.get('/ping', async (_req, res) => {
 
 })
 
+// This is designed as a Rest endpoint, although the HR api is GraphQL. I'm not sure which
+// is best for this, but presumably this slack api is a Backends for Frontends type Api, in
+// which case Rest, or Rpc is probably going to be more suitable that GraphQl.
+app.get('/:userId/manager', async (req, res) => {
```

Comment:
I *think* I would expect this service to implement the slack /command callback (it would receive a POST request with urlencoded body as described in https://api.slack.com/interactivity/slash-commands#app_command_handling), which would be exposed to the internet. I would expect slack to talk directly to this service, without another intermediate translation layer (other than whatever dapr/ingress routing we're using to expose it).

It seems reasonable to do the slack command implementation in a separate PR though.

Reply:
Yeah I had no idea how the slack integration was going to work, so thanks for the info, that will be something useful to work on next.